tests/unit/expire.tcl

```diff
@@ -603,9 +603,9 @@ start_server {tags {"expire"}} {
     } {-1}
 
     test {GETEX use of PERSIST option should remove TTL after loadaof} {
+       r config set appendonly yes
```

Comment:
this config is already set by a previous test, so unless we run just that test (with `--only`) it doesn't matter.
am i missing anything?

Reply:
yes, so the test is ok. I just thought adding it would make this test more self-contained?

Reply:
never mind.. just wanted to know